.github/workflows/test.yml

```diff
@@ -162,3 +162,17 @@ jobs:
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
       - name: Image digest
         run: echo ${{ steps.docker_build.outputs.digest }}
+
+  autodeploy:
+    runs-on: ubuntu-latest
+    needs: [docker-publish]
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Deploy Staging
+      if: github.ref == 'refs/heads/main'
```

Comment:
As mentioned in https://github.com/gnosis/safe-config-service/pull/451#discussion_r827891120

The following can be removed:

```suggestion
```